.github/workflows/deploy.yaml

```diff
@@ -25,7 +42,8 @@ jobs:
           - name: Production
     environment: ${{ matrix.environment.name }}
     concurrency:
-      group: ${{ github.workflow }}-${{ github.job }}-${{ matrix.environment.name }}
+      group: ${{ github.workflow }}-synchronize-${{ matrix.environment.name }}-${{ needs.prepare.outputs.component-name }}
```

Comment:
`github.job` is only available in context of `steps` so hard code it here to `synchronize`